src/config.ts

```diff
@@ -43,6 +43,10 @@ function onErrorRetry(
     return
   }
 
+  if (config.errorRetryCount && opts.retryCount > config.errorRetryCount) {
```

Comment:
errorRetryCount = 0, nerver return

Reply:
good catch, I'll make a PR for it later in the day, thanks.